aws/app/lambda/templates/templates.js

```diff
@@ -73,24 +77,54 @@ exports.handler = async function (event) {
       }
       break;
     case "GET":
+      SQL = "SELECT id, json_config, organization FROM Templates";
       // Get a specific form if given the id, all forms if not
       if (formID) {
-        SQL = !process.env.AWS_SAM_LOCAL
-            ? "SELECT id, json_config, organization FROM Templates WHERE id = :formID"
-            : "SELECT id, json_config, organization FROM Templates WHERE id = ($1)";
-        parameters = !process.env.AWS_SAM_LOCAL
-            ? [
+        SQL += !process.env.AWS_SAM_LOCAL
+            ? " WHERE id = :formID"
+            : " WHERE id = ($1)";
+
+        !process.env.AWS_SAM_LOCAL
+            ? parameters.push (
               {
                 name: "formID",
                 value: {
                   longValue: formID,
                 },
+              }) 
+            : parameters.push(formID);
+      }
+      if (limit) {
+        SQL += !process.env.AWS_SAM_LOCAL
+          ? " LIMIT :limit"
+          : " LIMIT ($1)";
+
+        !process.env.AWS_SAM_LOCAL
+          ? parameters.push (
+            {
+              name: "limit",
+              value: {
+                longValue: limit,
               },
-            ]
-            : [formID];
-      } else {
-        SQL = "SELECT id, json_config, organization FROM Templates";
+            }) 
+          : parameters.push(limit);
       }
+      if (offset) {
+        SQL += !process.env.AWS_SAM_LOCAL
+          ? " OFFSET :offset"
+          : " OFFSET ($2)";
```

Comment:
The way parameters are implemented above feels too permissive. Even though we have full control on the client side it might be relevant to allow for either a `formID` or a `limit` + `offset` type of request. 
For instance: if we send a `formID` and a `limit` we would end up having a SQL statement with two `($1)`  parameters.

Reply:
Great point. If there's a `formID`, then there's no need to also allow for `limit` or `offset`.

Reply:
I've added a `break` so that limit and offset won't be added to the SQL statement if there is a formID.

Reply:
It would be better to do this instead `if(!formID && offset)` rather than add breaks. It makes it difficult to read what is actually executing because you have to not only account for the if statements when reading the code but also the switch case statement